app/views/works/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<% if @work.errors.messages.any? %>
+  <section class="error-messages">
+    There were some issues with your submission
+    <ul>
+      <% @work.errors.messages.each do |column, problem_list| %>
+        <% problem_list.each do |problem| %>
+          <li>
+            <%= column %>: <%= problem %>
+          </li>
+        <% end %>
+      <% end %>
+    </ul>
+  </section>
+<% end %>
+
+<%= form_with model: @work, class: 'work-form' do |f| %>
+  
+
+  <div class="form-group">
+    <%= f.label :category %>
+  <%= f.select :category, options_for_select(["Album", "Book", "Movie"]), class: "form-control" %>
```

Comment:
Something is going wrong when you select on this form. The category for a new work isn't being recorded on the record in the database, so you can't get to the new work from any of the index pages. I believe that this line should read: 

`<%= f.select :category, ["album", "book", "movie"], {}, class: "form-control" %>`